ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/KsqlVersionChecker.java

```diff
@@ -53,13 +51,15 @@ public KsqlVersionChecker(
   }
 
   @Override
-  protected ZkClientProvider zkClientProvider() {
-    //This is used when collecting metrics in a kafka topic. Since KSQL isn't aware of ZK, we are
-    // returning null here and also turning off topic metrics collection in
-    // KsqlVersionCheckerConfig.
```

Comment:
Maybe we should keep part of the comment?

Reply:
@ijuma Thanks for the review, put back the comment.